client/src/components/RatingsComponents/ReviewsSidebar/RatingBreakdown.jsx

```diff
@@ -1,17 +1,34 @@
-import React from 'react';
+import React, {useState} from 'react';
 import StarBar from './StarBar.jsx';
 
 
-function RatingBreakdown ({ratings}) {
+function RatingBreakdown ({ratings, starsClicked, setStarsClicked}) {
   let starsCounts = ['1', '2', '3', '4', '5'];
+  const starClickedStyles = {
+    'textDecoration': 'underline'
+  };
+  const starUnClickStyles = {
+    'textDecoration': 'none'
+  };
+
   return (
     <div className='stars-breakdown-container'>
       {starsCounts.map((star) => {
         return (
           <div key={star} className='stars-bar-container'>
-            {/* <p className='stars-bar-label' key={star}>{`${star}`}</p>
-            <p className='stars-bar-label'>{`stars`}</p> */}
-            <a className='stars-bar-label' key={star}>{`${star}`}<span className='star-span'>stars</span></a>
+            <a className='stars-bar-label'
+              onClick={(e) => {
+                setStarsClicked(prevState => ({
+                  ...prevState,
```

Comment:
Getting another Parsing Error about this line; if you ain't getting it, disrgeard!

![image](https://user-images.githubusercontent.com/85661360/147869091-30d74dcd-8662-45ee-9e8a-49a8661feb30.png)

Reply:
eslint is mad about the spread operator. just ignore this. ill have to do more research to figure out a way to configure eslint to handle all new stuff from es7 and beyond.